create-presigned-download-url/src/test/java/no/unit/nva/download/publication/file/aws/s3/AwsS3ServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package no.unit.nva.download.publication.file.aws.s3;
+
+import com.amazonaws.HttpMethod;
+import com.amazonaws.SdkClientException;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.GeneratePresignedUrlRequest;
+import no.unit.nva.download.publication.file.aws.s3.exception.S3ServiceException;
+import no.unit.nva.download.publication.file.publication.RestPublicationService;
+import no.unit.nva.download.publication.file.publication.exception.NoResponseException;
+import nva.commons.exceptions.ApiGatewayException;
+import nva.commons.utils.Environment;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class AwsS3ServiceTest {
+
+    public static final String REGION = "region";
+    public static final String BUCKET_NAME = "bucketname";
+    public static final String OBJECT_KEY = "12345";
+    public static final String PRESIGNED_DOWNLOAD_URL = "https://example.com/download/12345";
+    public static final String MIME_TYPE_APPLICATION_PDF = "application/pdf";
+
+    private AmazonS3 s3Client;
+    private Environment environment;
+
+    /**
+     * Set up environment.
+     */
+    @BeforeEach
+    public void setUp() {
+        s3Client = mock(AmazonS3.class);
+        environment = mock(Environment.class);
+    }
+
+    @Test
+    @DisplayName("calling Constructor With All Env")
+    public void callingConstructorWithAllEnv() {
```

Comment:
Missing description of expected behavior: calling Constuctor with All Env does what?